### ISSUE TITLE
Telstra - Our coverage maps

### ISSUE THREAD

**Issue by gyCfjSnO** [AdAway/adaway.github.io#370]
### Which domain(s) should be unblocked?

`cdnssl.clicktale.net`

### Why should the domain(s) be unblocked?

When the domain `cdnssl.clicktale.net` is blocked you are unable to view the coverage map overlay at https://www.telstra.com.au/coverage-networks/our-coverage

This is what it should look like:

![image](https://github.com/AdAway/adaway.github.io/assets/49738152/4c39ece0-b40b-4f62-9f9c-fbda0a1126b3)

**Comment by jawz101** [comment]
The site probably shouldn't be dependent on 3rd party, user behavior analysis code to hinder its primary functions.  Regardless, 99.9999% of the people who visit the site are not likely blocking the domain so I don't see it as a big hinderance.  Still, I would imagine it would be just as easy to make it conditional - i.e. if clicktale can't run, skip trying to run it and execute the remaining code.

**Comment by gyCfjSnO** [comment]
> The site probably shouldn't be dependent on 3rd party, user behavior analysis code to hinder its primary functions. Regardless, 99.9999% of the people who visit the site are not likely blocking the domain so I don't see it as a big hinderance. Still, I would imagine it would be just as easy to make it conditional - i.e. if clicktale can't run, skip trying to run it and execute the remaining code.

I am increasingly frustrated by the fact that many prominent Australian companies rely on third-party providers to deliver important content. This content is often blocked by most blocklist maintainers.

**Comment by jawz101** [comment]
Yeah.  One thing I do is look at the 3rd party's site to see what is the intention of their product.  E.g. case studies, customer testimonials.

In this instance, clicktale's purpose is to monitor what a user clicks upon to measure interest.  Which may benefit the user I suppose if everyone is clicking on their locality, it says to the business "we need to put up more cell towers there."
However, mostly the case studies are targeted towards the business owner to say "we increased our revenue by 30 million last month because we redesigned our site to put splurge purchases in key spots".  Personally, I want my interests to be my own and not give company's extra information to nickel and dime every amount of money they can get out of my visit to their store.

Every time you walk into an electronics store a person hovers around you to answer any questions, suggest certain brands, and upsell accessory purchases.  The premise online isn't anything new.  If I have the choice to say 'no thank you' I'd prefer to make that choice.  Ultimately, I can whitelist that domain if I really want to use it.

As far as 3rd party code, I see it both from my perspective and the company's.  As a user, I don't like that the company has put trust in another company's code.  What if that 3rd party code additionally siphons off your interactions to drive another product of theirs to sell to other clients?  What if that 3rd party company's product is compromised and it hosts some malicious code?

On the other hand, it's software.  I don't expect every site developer to reinvent the wheel and make their own, sophisticated click tracker.  Eventually it gets out of hand though and you visit some sites with 30+ company's code driving their whole site.  You block all but the sites own code and the site's primary functions still work.  This is, in the least, how I think it should be.  That phone company shouldn't design that coverage map to break because the code to analyze visit clicks doesn't fire.  That's just my opinion.

And if I was still the list maintainer, I'd probably remove it from the list if it wasn't used within mobile app 3rd party code. I wanted AdAway to just be representative of mobile app embedded code traffic.  I'll leave it to every other blocklist to block website junk.  Browser ad blockers can make fancier rules and they have way more people scrutinizing them.  I originally asked to maintain the list because the AdAway list was old and crusty.  It had a few hundred entries and most of the company domains no longer existed.  I knew there were thousands of mobile ad and tracking domains out there that had sprung up and every year new ad company's start up while others shut down.  Most of my commits were to remove entries I'd originally added once more people were using my personal list- so it was nice to have the feedback.
